app/features/accessibility.test.js

```diff
@@ -0,0 +1,60 @@
+import test from 'ava'
+import { setupPptrTab, teardownPptrTab, getActiveTool, changeMode } from '../../tests/helpers'
+
+test.beforeEach(setupPptrTab)
+
+const contrastValueSelector = `document.querySelector('visbug-ally').$shadow.querySelector("span[contrast]").textContent.trim()`
+
+test('Can be activated', async t => {
+  const {page} = t.context;
+  await changeMode({page, tool: 'accessibility'})
+
+  t.is(await getActiveTool(page), 'accessibility')
+  t.pass()
+})
+
+test('Can reveal color contrasts between html nodes and backgrounds', async t => {
+  const {page} = t.context;
+  await changeMode({page, tool: 'accessibility'})
+
+  await page.hover('.google-blue')
+  const blueContrastValue = await page.evaluate(contrastValueSelector)
+  t.is(blueContrastValue, "3.56")
+  t.pass()
+
+
+  await page.hover('.google-red')
+  const redContrastValue = await page.evaluate(contrastValueSelector)
+  t.is(redContrastValue, "4.29")
+  t.pass()
+
+  await page.hover('.google-yellow')
+  const yellowContrastValue = await page.evaluate(contrastValueSelector)
+  t.is(yellowContrastValue, "1.84")
+  t.pass()
+})
+
+test('Does not show a11y tooltip on <svg> node', async t => {
+  const {page} = t.context;
+  await changeMode({page, tool: 'accessibility'})
+
+  await page.mouse.click(175, 80) // an empty space of the first svg element
```

Comment:
the page click tests passed though, so maybe that's a more stable method to use for these tests?